src/service/BookingService.java

```diff
@@ -83,4 +94,39 @@ public int modifyBooking (BookingDataDTO bookingDataDTO) {
 	public int deleteBooking (int id){
 		return bookingDataDAO.delete(id);
 	}
+	
+	public List<GuestDataDTO> loadGuestList() {
+		return guestDataDAO.searchGuestList();
+	}
+	
+	public GuestDataDTO loadGuestById(int idSearch) {
```

Comment:
We can't return a DTO here, we're going to need to create a GuestData class within the service package, and implement the conversion pattern.